Automattic/newspack-plugin
# Display latest releases info in plugin dashboard

### All Submissions:






### Changes proposed in this Pull Request:



Adds a "What's New" card in the plugin dashboard, displaying the list of latest changes to the main components of the Newspack suite:

![image](https://user-images.githubusercontent.com/7383192/84502957-98296b80-acb9-11ea-8290-b6d561be1ea3.png)

### How to test the changes in this Pull Request:

1. Build the JS
2. Visit the dashboard
3. Observe a new card at the top

Note these these requests to GH are rate-limited:

> For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the user making requests.

### Other information:
